doc_requirements.txt

```diff
@@ -0,0 +1,3 @@
+Sphinx==4.4.0
+pydata-sphinx-theme==0.7.1
```

Comment:
The latex version (0.8.0) doesn't work well with older versions of Jinja (Sphinx dependency). xref https://github.com/pydata/pydata-sphinx-theme/issues/564. So, pin to the older version for now.